src/components/generic/Button/Button.jsx

```diff
@@ -0,0 +1,89 @@
+import React, {JSX} from 'react';
+import PropTypes from 'prop-types';
+import {Button as MomentumButton} from '@momentum-ui/react';
+import webexComponentClasses from '../../helpers';
+import Icon from '../Icon/Icon';
+
+/**
+ * Button component
+ *
+ * @param {object} props  Data passed to the component
+ * @param {JSX.Element} props.children  List of children
+ * @param {string} [props.className]  Custom CSS class to apply
+ * @param {string} props.icon  Name of the icon to be rendered on the old button
+ * @param {string} props.iconColor  Color of the icon
+ * @param {boolean} props.isDisabled  Flag indicating button disabled
+ * @param {Function} props.onClick  OnClick callback
+ * @param {boolean} props.showText  Flag that indicates whether to display text on the old button
+ * @param {object} props.style  Inline style object for the component
+ * @param {string} props.text  Text to be disaplyed on old buttons
+ * @param {string} props.title  Tooltip to be displayed
+ * @param {string} props.tooltip  A11y tooltip
+ * @param {'default'|'join'|'cancel'|'ghost'} [props.type='default']  Button type
+ * @returns {object}  JSX of the element
+ */
+export default function Button({
+  children,
+  className,
+  icon,
+  iconColor,
+  isDisabled,
+  onClick,
+  title,
+  showText,
+  style,
+  text,
+  tooltip,
+  type,
+}) {
+  const cssClasses = webexComponentClasses('button', className, {[`-${type}`]: true});
```

Comment:
I think it would be ok to just use `[type]: true`, without an extra dash, it looks complicated for no particular benefit

Reply:
actually, leave it like this, the vars look better with a double dash in the theme scss files